File: py/232.py
class MyQueue:

    def __init__(self):
        """
        Initialize your data structure here.
        """
        self._list = []

    def push(self, x):
        """
        Push element x to the back of queue.
        :type x: int
        :rtype: void
        """
        self._list.append(x)

    def pop(self):
        """
        Removes the element from in front of queue and returns that element.
        :rtype: int
        """
        if len(self._list) > 0:
            r = self._list[0]
            self._list.remove(self._list[0])
            return r

    def peek(self):
        """
        Get the front element.
        :rtype: int
        """
        return self._list[0]

    def empty(self):
        """
        Returns whether the queue is empty.
        :rtype: bool
        """
        return len(self._list) == 0

File: js/232.js
/**
 * Initialize your data structure here.
 */
var MyQueue = function() {
  this._stack = []
};

/**
 * Push element x to the back of queue. 
 * @param {number} x
 * @return {void}
 */
MyQueue.prototype.push = function(x) {
    
  this._stack.push(x)
};

/**
 * Removes the element from in front of queue and returns that element.
 * @return {number}
 */
MyQueue.prototype.pop = function() {
  return this._stack.shift(x)
};

/**
 * Get the front element.
 * @return {number}
 */
MyQueue.prototype.peek = function() {
  return this._stack[0]
};

/**
 * Returns whether the queue is empty.
 * @return {boolean}
 */
MyQueue.prototype.empty = function() {
  return this._stack.length === 0
};